packages/pagination/src/hooks/useBreakpoint.hook.ts

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
```

Comment:
When you say this is SSR + SSG compatible what do you mean? `uesEffect` wont run on the server so the first render will have no breakpoint data. 
Can i suggest we set `isMobile` as default true. As this is where most of our customers are and where and extra render or layout shift will cause the most stress for the  the user

Reply:
That's a good point. The default is now isMobile as true, thanks